app/scripts/lib/fxa-client.js

```diff
@@ -235,29 +241,32 @@ function (FxaClient, $, p, Session, AuthErrors) {
               });
     },
 
-    passwordReset: function (originalEmail, relier) {
-      var self = this;
+    passwordReset: function (originalEmail, relier, options) {
       var email = trim(originalEmail);
+      options = options || {};
 
       return this._getClient()
-              .then(function (client) {
-                var clientOptions = {
-                  service: relier.get('service'),
-                  redirectTo: relier.get('redirectTo'),
-                  resume: self._createResumeToken(relier)
-                };
+        .then(function (client) {
+          var clientOptions = {
+            service: relier.get('service'),
+            redirectTo: relier.get('redirectTo')
+          };
 
-                return client.passwordForgotSendCode(email, clientOptions);
-              })
-              .then(function (result) {
-                Session.clear();
-                return result;
-              });
+          if (options.resume) {
+            clientOptions.resume = options.resume;
+          }
+
+          return client.passwordForgotSendCode(email, clientOptions);
+        })
+        .then(function (result) {
+          Session.clear();
```

Comment:
See https://github.com/mozilla/fxa-content-server/issues/2666. I'd like this Session.clear to be removed from fxa-client.js, but that's for a separate PR.